src/components/interactive-map.js

```diff
@@ -101,16 +143,21 @@ export default class InteractiveMap extends PureComponent {
   }
 
   componentDidMount() {
-    // Register event handlers
     const {eventCanvas} = this.refs;
     const {mapControls} = this.props;
 
+    // Register event handlers defined by map controls
     const events = {};
     mapControls.events.forEach(eventName => {
       events[eventName] = this._handleEvent;
     });
 
-    this._eventManager = new EventManager(eventCanvas, {events});
+    const eventManager = new EventManager(eventCanvas, {events});
+
+    // Register event handlers for click and hover
+    eventManager.on('mousemove', this._onMouseMove);
```

Comment:
Will this override these events, or add additional events? Could be good to comment for those of use who are not `EventManager` experts.

If override, I guess we could just use Object.assign on the `{events}`.